source/patterns/@aws-solutions-constructs/core/test/utils.test.ts

```diff
@@ -142,4 +143,42 @@ test('Test consolidate props with one arg', () => {
 
   expect(result).toEqual(arg1);
 
-});
\ No newline at end of file
+});
+
+test('Test generateName sunny day for current construct with unefined name argument', () => {
+  const stack = new cdk.Stack(undefined, "some-new-id");
+
+  const newName = defaults.generateName(stack);
+  expect(newName.length).toBeGreaterThan(5);
```

Comment:
Just so I understand, why is the number 5 significant here?

Reply:
It's a bit random - just making that it is set to something and not get fooled by a newline or something like that